### PR TITLE
[experimental] added new parameter types to `ldscsim`

### PR DESCRIPTION
**Pull request by nikbaya**

### CODE REVIEW

**Comment by tpoterba** [comment]
you have a doctest failure:
```

=================================== FAILURES ===================================
______________ [doctest] hail.experimental.ldscsim.get_cov_matrix ______________
034     trait 1 & trait 4 :math:`r_g` = 0
035     trait 2 & trait 3 :math:`r_g` = 0.9
036     trait 2 & trait 4 :math:`r_g` = 0.15
037     trait 3 & trait 4 :math:`r_g` = 1
038     To obtain the covariance matrix corresponding to this scenario :math:`h^2` values are
039     ordered according to user specification and :math:`r_g` values are ordered by the 
040     order in which the corresponding genetic covariance terms will appear in the 
041     covariance matrix, reading lines in the upper triangular matrix from left to
042     right, top to bottom (read first row left to right, read second row left to 
043     right, etc.), exluding the diagonal.
Differences (unified diff with -expected +actual):
    @@ -1,4 +1,12 @@
    -array([[0.1       , 0.06928203, 0.09899495, 0.        ],
    -       [0.06928203, 0.3       , 0.22045408, 0.06363961],
    -       [0.09899495, 0.22045408, 0.2       , 0.34641016],
    -       [0.        , 0.06363961, 0.34641016, 0.6       ]])
    +covariance matrix is not positive semidefinite.
    +adjusting rg values to make covariance matrix positive semidefinite
    +0.4 -> 0.42023852645344634
    +0.7 -> 0.5623351779264535
    +0.0 -> 0.04812102869845767
    +0.9 -> 0.7179252549815345
    +0.15 -> 0.20103397396043554
    +1.0 -> 0.7534523335539473
    +(array([[0.1       , 0.07278745, 0.0795262 , 0.0117872 ],
    +       [0.07278745, 0.3       , 0.17585505, 0.08529149],
    +       [0.0795262 , 0.17585505, 0.2       , 0.26100354],
    +       [0.0117872 , 0.08529149, 0.26100354, 0.6       ]]), [0.42023852645344634, 0.5623351779264535, 0.04812102869845767, 0.7179252549815345, 0.20103397396043554, 0.7534523335539473])

```

And you'll need to update `ldscsim.rst` because you changed the list of exposed functions, it looks like.

**Comment by danking** [comment]
I've authorized another test of this PR.

**Comment by danking** [comment]
Taking ownership since Tim is ooo.

**Comment by tpoterba** [comment]
now failing to import `create_cov_matrix`. looks like you removed that? Should be removed from the docs too.

You can build the docs locally with `make -C hail docs`

**Comment by nikbaya** [comment]
I renamed `create_cov_matrix` to `get_cov_matrix` and forgot to edit `index.rst` to reflect the change. `index.rst` should be fixed now.

**Comment by tpoterba** [comment]
👍 testing now

File: hail/python/hail/experimental/ldscsim.py [hunk 1]
```diff
@@ -268,23 +289,35 @@ def multitrait_ss(mt, h2, pi, rg=0, seed=None):
     Warning
     -------
     May give inaccurate results if chosen parameters make the covariance matrix 
-    not positive semi-definite.
+    not positive semi-definite. Covariance matrix is likely to not be positive
+    semi-definite when :mat:`p_{TT}` is small and rg is large.
```
Comment (tpoterba) [inline hail/python/hail/experimental/ldscsim.py]: typo: this is causing a docs build failure.

**Comment by tpoterba** [comment]
Getting more failures :(

Let's sit down tomorrow and I'll show you how to build docs locally

**Pull request merged**